source/includes/_tipser-sdk.md

```diff
@@ -59,6 +51,209 @@ Initialized Tipser SDK object that can be used to perform API calls further in t
+The example starts Tipser SDK instance in which Tipser shop has the primary color set to yellow.
+
+For configuration options see the [configuration](#configuration-options) chapter of our documentation.
+
+##Customization of Tipser environment
+
+By default, Tipser SDK connects to the production Tipser environment. Yet, if the testing environment is preferred (e.g. in order to do **test purchase**), then it can be customized with env parameter in TipserSDK options, which accepts the following values:
+
+`prod` or production
+
+`stage` or staging
+
+`dev` or development
+
+
+*Example:*
+
+```javascript
+var tipser = TipserSDK("5aa12d639d25800ff0e56fc5", {env: "stage"});
+```
+
+
+##The script API
+
+`productId`: string passed in product-related API calls. It consists of the **masterProductId** and, if the product has variants, the **variantId**, concatenated with an **underscore "_"**. Then the **productId** should look like this: 
+
+```
+<masterProductId>_<variantId>
+```
+
+`posData` string passed in the `options` is returned with other order fields from Tipser commissions API. 
+<aside class="warning">Warning: the maximal number of characters in **posData** is limited to 4000 characters and will be truncated if a longer string is passed.</aside>
+
+
+##Adding product to cart
+
+```javascript
+addToCart(productId: string, options: {posData?: string}): Promise<void>
+```
+
+Adds a product with a given productId (specific to Tipser) to a cart.
+
+`productId`: string, with the stucture described [here](#the-script-api).
+
+`options`: currently the only supported option is [posData](#the-script-api), which is a string to be attached to a product order. 
+
+*Returns*:
+
+A Promise that resolves to no value but gives a chance to check if the request was successful or not (using `.then()/.catch()` semantics)
+
+*Example*:
+
+```javascript
+tipser.addToCart("5a98be67c0bdfb0c30865609_5a98be67c0bdfb0c30866509")
+  .then(() => console.log("successfuly added to cart"))
+  .catch((err) => console.log("failed to add to cart", err))
+```
+
+##Opening product dialogue
+
+```javascript
+openProductDialog(productId: string): void
+```
+
+Opens Tipser product dialog for a product with given tipser product id.
+
+*Example*:
+
+```javascript
+tipser.openProductDialog("5a16534aa5af9b3af450c33d_5a16537da5af9b3af450c33f");
+```
+
+
+##Opening purchase dialog
+
+```javascript
+openPurchaseDialog(): void
+```
+
+Opens Tipser purchase (checkout) dialog with products that are currently in the shopping cart.
 
-For configuration options see [configuration](#configuration-options) chapter of our documentation.
+
+##Getting cart size:
+
+```javascript
+getCurrentCartSize(): Promise<number>;
+```
+
+Returns current Tipser cart size (including items added to cart from other shops). That value can change over time, so it may be necessary to call it repeatedly (polling) to have up-to-date cart size all the time.
```

Comment:
I so it may be necessary to call it repeatedly (polling) -> so it may be necessary to call it repeatedly (e.g. on every page load or polling with a certain interval)